packages/atomic/create-atomic/scripts/packageTemplate.json

```diff
@@ -3,6 +3,11 @@
   "description": "{{titleCase project}} project",
   "version": "0.1.0",
   "private": true,
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "collection": "dist/collection/collection-manifest.json",
+  "collection:main": "dist/index.js",
```

Comment:
Once you have dist/dist-custom-element output targets, Stencil will instruct the user to update their package.json file